src/video_core/renderer_vulkan/vk_swapchain.h

```diff
@@ -34,8 +34,8 @@ class VKSwapchain {
     void Present(VkSemaphore render_semaphore);
 
     /// Returns true when the swapchain needs to be recreated.
-    bool NeedsRecreation(bool is_srgb) const {
-        return HasColorSpaceChanged(is_srgb) || IsSubOptimal() || NeedsPresentModeUpdate();
+    bool NeedsRecreation(bool is_srgb, Layout::FramebufferLayout layout) const {
```

Comment:
```suggestion
    bool NeedsRecreation(bool is_srgb, const Layout::FramebufferLayout& layout) const {
```

Since this isn't a small object